CRT.c

```diff
@@ -69,6 +69,14 @@ bool CRT_utf8 = false;
 
 const char *const *CRT_treeStr = CRT_treeStrAscii;
 
+const char* CRT_degreeSign() {
+   #ifdef HAVE_LIBNCURSESW
+   if (CRT_utf8)
+      return "\xc2\xb0";
+   #endif
```

Comment:
With libcurses you can still use ° as it is within most non-UTF8 charsets (ISO-8859-x), even CP437, have it at 0x90.

Reply:
@fasterit can you take a look at the added commit